neuromation/api/jobs.py

```diff
@@ -137,6 +137,7 @@ def __init__(self, core: _Core, config: Config, parse: Parser) -> None:
         description: Optional[str] = None,
         is_preemptible: bool = False,
         schedule_timeout: Optional[float] = None,
+        life_span: timedelta = timedelta.max,
```

Comment:
Please use `Optional[float]` type here, like for `schedule_timeout`. `None` means *no lifespan limit`.
`float` means *time in seconds*, not *amount of minutes*.

Reply:
no difference, will do.

Reply:
(I thought `timedelta` is the most convenient way to store time intervals, but ok will do in seconds, `float`)